### PR TITLE
make the element private,rename to value.

### PR DESCRIPTION
**Pull request by VivianWenhuiLiang**

### CODE REVIEW

File: src/com/vivian/java/games/pushingbox/PushingBox.java [hunk 1]
```diff
@@ -8,16 +8,16 @@
 import java.io.IOException;
 
 public class PushingBox {
-    public enum Element {
+    private enum Element {
         BLANK(0), PLAYER(1), BOX(2), DESTINATION(3), OBSTACLE(4);
-        int el;
+        int value;
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/games/pushingbox/PushingBox.java]: I mean, make the instance variable "value" private.

File: src/com/vivian/java/games/pushingbox/PushingBox.java [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
 
-        Element(int el) {
-            this.el = el;
+        Element(int value) {
+            this.value = value;
         }
 
         public int getEl() {
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/games/pushingbox/PushingBox.java]: Rename the method accordingly.

**Review by GeorgeZhiXu — COMMENTED** [review]
Have you changed the type of state and destination yet?

File: src/com/vivian/java/games/pushingbox/PushingBox.java (hunk 1 above)
Comment (VivianWenhuiLiang) [reply src/com/vivian/java/games/pushingbox/PushingBox.java]: fixed

File: src/com/vivian/java/games/pushingbox/PushingBox.java (hunk 2 above)
Comment (VivianWenhuiLiang) [reply src/com/vivian/java/games/pushingbox/PushingBox.java]: fixed

**Review by GeorgeZhiXu — APPROVED** [review]

**Pull request merged**